roq-frontmatter/deployment/src/main/java/io/quarkiverse/roq/frontmatter/deployment/RoqFrontMatterProcessor.java

```diff
@@ -93,10 +94,20 @@ void bindEndpoints(
             RoqFrontMatterConfig config,
             BuildProducer<SelectedPathBuildItem> selectedPathProducer,
             BuildProducer<NotFoundPageDisplayableEndpointBuildItem> notFoundPageDisplayableEndpointProducer,
-            RoqFrontMatterOutputBuildItem roqOutput) {
+            RoqFrontMatterOutputBuildItem roqOutput,
+            List<RoqFrontMatterAliasesBuildItem> aliases) {
         if (roqOutput == null) {
             return;
         }
+
+        if (!aliases.isEmpty()) {
+            for (RoqFrontMatterAliasesBuildItem aliasItem : aliases) {
+                for (String alias : aliasItem.aliases()) {
+                    selectedPathProducer.produce(new SelectedPathBuildItem(prefixWithSlash(alias), null));
```

Comment:
The prefix won't be necessary anymore as it will be added when using Link (see next comment)